src/Mobs/Monster.cpp

```diff
@@ -435,19 +428,41 @@ void cMonster::CalcLeashActions()
 			SetLeashToPos(nullptr);
 		}
 	}
-	else if (IsLeashed())  // Mob is already leashed to an entity: follow it.
+
+	if (!IsLeashed())
+	{
+		return;
+	}
+
+	static const double LeashNaturalLength = 5.0;   // Length at equilibrium
+	static const double LeashMaximumLength = 10.0;  // Length where the leash breaks
+	static const double LeashSpringConstant = 20.0;  // How elastic the leash is
```

Comment:
Similarly here, a higher value means less elastic, which is not the intuiitive meaning of the comment.

Reply:
I'm struggling to come think of language that isn't technical. It makes sense to me because the spring constant is proportional to the [elastic modulus](https://en.wikipedia.org/wiki/Young%27s_modulus).

Would "stiff" make more sense to you?